Album.html

```diff
@@ -46,24 +66,42 @@
     </div>
```

Comment:
In this case we don't need to use forms and inputs to add button. Please follow the simple way - using divs to make the button element.

Reply:
Done